packages/gatsby/src/schema/__tests__/__snapshots__/rebuild-schema.js.snap

```diff
@@ -1290,6 +1307,53 @@ enum SitePageFieldsEnum {
   componentChunkName
   matchPath
   pageContext
+  pluginCreator___resolve
```

Comment:
yikes! 😱

Reply:
Yeah, it is a part of public type definition:

https://github.com/gatsbyjs/gatsby/blob/08c5b2a55559fd1419222ae6cb5a7d3c1e2a098d/packages/gatsby/src/schema/types/built-in-types.ts#L114-L116